gistit.py

```diff
@@ -335,3 +338,18 @@ def test_token(self):
 
     def test_create(self):
         self.assertReadmeContainsOutput(['create', '-h'])
+
+class SimpleRun(unittest.TestCase):
+    def setUp(self):
+        cmd_args = [sys.executable, 'gistit.py', '--help']
+        self.help_output_lines = subprocess.Popen(
+            cmd_args,
+            stdout=subprocess.PIPE).communicate()[0].decode('utf-8')
+
+    def test_no_args(self):
+        cmd_args = [sys.executable, 'gistit.py']
+        self.test_output_lines = subprocess.Popen(
+            cmd_args,
+            stdout=subprocess.PIPE).communicate()[0].decode('utf-8')
+        self.assertTrue(self.help_output_lines == self.test_output_lines)
```

Comment:
Use `assertEqual` instead, for a better error message.

Reply:
Updated